installer/resources/pacbot_app/files/DB.sql

```diff
@@ -2954,4 +2954,6 @@ END $$
 
 DELIMITER ;
 
-CALL update_filter_for_tag(@MANDATORY_TAGS);
\ No newline at end of file
+CALL update_filter_for_tag(@MANDATORY_TAGS);
+
+delete from pac_v2_ui_options where optionName in ('Application','Environment','Product','Owner');
```

Comment:
This won't remove if the mandatory tags are set to some other value
Instead we can have- delete from pac_v2_ui_options where optionValue like '%.tags.%';